### PR TITLE
adding weights to the challenges

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**

### CODE REVIEW

File: db/migrate/20200505093714_add_weight_to_challenge.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddWeightToChallenge < ActiveRecord::Migration[5.2]
+  def change
+    add_column :challenges, :weight, :integer, default: 0
```
Comment (piotrekpasciak) [inline db/migrate/20200505093714_add_weight_to_challenge.rb:3]: Generally, it's good idea to add `null: false` together with `default` option.

Comment (nikhilrayaprolu) [reply db/migrate/20200505093714_add_weight_to_challenge.rb:3]: made a new migration reflecting the same

Comment (alkeshrails) [reply db/migrate/20200505093714_add_weight_to_challenge.rb:3]: @nikhilrayaprolu This is open PR, so you can rollback migration and create a single migration with column type: float, default 0 and null false

Comment (alkeshrails) [reply db/migrate/20200505093714_add_weight_to_challenge.rb:3]: Please pull schema.rb as well

**Pull request merged**